src/domains.js

```diff
@@ -24,6 +24,10 @@ export function getPayPalDomain() : string {
     if (__ENV__ === ENV.SANDBOX) {
         return `${ getProtocol() }://www.sandbox.paypal.com`;
     }
+    
+    if (__ENV__ === ENV.SANDBOX_V2) {
+        return `${ getProtocol() }://www.v2.sandbox.paypal.com`;
```

Comment:
This is supposed to be `www-v2.sandbox.paypal.com`

Reply:
I actually wonder if it's incorrect in the requirements.  Will check.

Reply:
I thought it was an error, since`www-v2` didn´t seem like an ivalid value, but just checked in Jira and you´re right. 

Changed!

Reply:
Verified with contact and he said it needs to be with the dash.